### ISSUE TITLE
video not work on iOS13

### ISSUE THREAD

**Issue by comeStudy** [mwaterfall/MWPhotoBrowser#649]
video not work on iOS13。MPMoviePlayerController is no longer available. Use AVPlayerViewController in AVKit.

**Comment by amith-ranasinghe** [comment]
I am facing the same issue. any workaround?

**Comment by ouhuiju** [comment]
Is this project out of maintenance？

**Comment by xmkaufman** [comment]
Yes - four years out of maintenance. See pull request 316 for switch from MPMoviePlayerController to AVPlayerViewController - https://github.com/mwaterfall/MWPhotoBrowser/pull/316

**Comment by nwkeeley** [comment]
Can anyone suggest any alternatives with similar functionality?

**Comment by MahmoudMMB** [comment]
You can use this. I solved the problem here.
https://github.com/MahmoudMMB/MWPhotoBrowser-AVKit

**Comment by manishvpatel** [comment]
Very much thank you @MahmoudMMB
